internal/component/azure/servicebus/subscription.go

```diff
@@ -148,7 +199,7 @@ func (s *Subscription) ReceiveAndBlock(handler HandlerFunc, lockRenewalInSec int
 		}
```

Comment:
(Leaving a comment here because I can't comment on L194)

There is a problem that when bulk subscribing is enabled, max active message doesn't work as expected. This is because we are taking a single entry from `s.activeMessageChan` for the entire set of bulk messages. So essentially rather than number of active messages, becomes number of active bulk messages. Not sure if that's intended.

Reply:
Hello @ItalyPaleAle, thanks for catching this, this has been updated now. 

Instead of `activeMessagesChan`, it's now `activeOperationsChan`. The channel is now used to control concurrent active operations, rather than messages. The size of `activeOperationsChan` is calculated as `maxActiveMessages/maxBulkCount`, where `maxBulkCount` can be `[1, maxActiveMessages]`

In case of normal subscribe, `maxBulkCount` is 1, so `activeOperationsChan` has the size `maxActiveMessages` (same as before). Here, each operation is just one receive request. 

In case of bulk subscribe, if `maxBulkCount` is 100 and `maxActiveMessages` is 10000, `activeOperationsChan`'s size is 100.

Let me know what you think about this!

/cc @mukundansundar @DeepanshuA

Reply:
Thanks for making this change @shubham1172 ! Looks good to me now :)